packages/core/src/schema/models/evaluations.ts

```diff
@@ -0,0 +1,29 @@
+import { relations } from 'drizzle-orm'
+import { bigint, bigserial, index, text, varchar } from 'drizzle-orm/pg-core'
+
+import { latitudeSchema, workspaces } from '..'
+import { timestamps } from '../schemaHelpers'
+
+export const evaluations = latitudeSchema.table(
+  'evaluations',
+  {
+    id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
+    name: varchar('name', { length: 256 }).notNull(),
+    description: text('description').notNull(),
+    prompt: text('prompt').notNull(),
+    workspaceId: bigint('workspace_id', { mode: 'number' })
+      .notNull()
+      .references(() => workspaces.id, { onDelete: 'cascade' }),
+    ...timestamps(),
+  },
+  (table) => ({
+    evaluationWorkspaceIdx: index('workspace_idx').on(table.workspaceId),
+  }),
+)
```

Comment:
**Evaluations** — The evaluations defined in the workspace.

`prompt` – Here's where the evaluation document is stored. Not sure if we should let the user define this directly or help them in some way. Also, should this be editable afterwards?

Reply:
Questions for #product

Reply:
Talked with product. We may want to add version control in the future, but not for now. This will just be a `prompt` field, and can be modified after its creation.